db/migrate/20180827103128_create_meals_ingredients.rb

```diff
@@ -0,0 +1,8 @@
+class CreateMealsIngredients < ActiveRecord::Migration[5.2]
+  def change
+    create_table :meals_ingredients do |t|
+
```

Comment:
Shouldn't it be a join table? We've got only one index here. I ask a lot of questions. That's not how reviews should be done x.x

Reply:
I believe it should, just like UsersAllergies.
It should probably have two links: `meal_id` and `ingredient_id`.

Reply:
yes, it should have 'meal' connection too